### ISSUE TITLE
Library crashes on Windows XP

### ISSUE THREAD

**Issue by tresf** [java-native/java-simple-serial-connector#9]
* @scream3r's `jSSC-2.8_x86.dll` - Does not crash Windows XP
* Our version - Crashes Windows XP

After some research I've found out this is due to the bundling (or lack thereof) of some standard libraries.

https://stackoverflow.com/a/6405064/3196753
https://stackoverflow.com/a/18138926/3196753
https://stackoverflow.com/a/12922118/3196753
https://sourceforge.net/p/mingw-w64/discussion/723797/thread/116e5fd5/

So far, this helps dependency walker (it stops complaining about `libstdc++`, etc) but the app still crashes.
```cmake
set(CMAKE_CXX_STANDARD_LIBRARIES "-static-libgcc -static-libstdc++ ${CMAKE_CXX_STANDARD_LIBRARIES}")
```

**Comment by tresf** [comment]
Ok, I think I got it... 

```diff
- set(CMAKE_CXX_STANDARD_LIBRARIES "-static-libgcc -static-libstdc++ ${CMAKE_CXX_STANDARD_LIBRARIES}")
+ set(CMAKE_CXX_STANDARD_LIBRARIES "-static-libgcc -static-libstdc++ -fPIC ${CMAKE_CXX_STANDARD_LIBRARIES}")
```

**Comment by tresf** [comment]
Closed via 0ba4018.
